### DISCUSSION TITLE
Support for rinse.fm and idaidaida.net

### DISCUSSION THREAD

**Discussion by joonas-foo** [web-scrobbler/web-scrobbler#4055] [Ideas]
Two sites that maybe could be supported by the webscrobbler, rinse.fm and idaidaida.net .

**Comment by joonas-foo** [comment]
Whoops! Noticed idaidaida.net uses Mixcloud for it's mixes. The webscrobbler could support https://www.bbc.co.uk/sounds ? What about adding support for internet radio streams to last.fm, there could be a top 1000 for the most listened streams!

**Comment by patrickzavadskis** [comment]
Support straight from idaidaida.ee could still be nice.
